flybywiresim/aircraft
# feat(a380x/fcu): add correct QFE label using baro preselect

Fixes #[issue_no]

## Summary of Changes


* add QFE label using baro preselect
* add characters q,f,e and test q to FCU font
* clean up old QFE label
* adjust the alignment and size of elements in the baro window
* edit the 7-segment digits/character in the FCU font to sit on the correct baseline
* edit styling of all FCU digits to match changes in new font scale

## Screenshots (if necessary)



![image](https://github.com/user-attachments/assets/681bacae-ed3e-4eb0-962f-6b176f547eb8)
![image](https://github.com/user-attachments/assets/b9c71e3d-5c36-4398-a4d3-86af561ce4ce)


## References


![image](https://github.com/user-attachments/assets/3f5b82d2-b2cf-4154-8dc1-973fcbbca6ba)
![image](https://github.com/user-attachments/assets/705c6e01-b5af-4281-a3a9-6086800ba31d)




## Additional context



Discord username (if different from GitHub): Heclak

## Testing instructions


1. Turn on ANN LT to test and compare to screenshot/reference above for light test mode.
2. Testing the QFE label would require the use of behaviour debug tool to trigger as there is no in-cockpit method to switch to QFE mode yet. You can set XMLVAR_Baro1_Mode to 0 to switch baro mode to 0.
3. Turn all the FCU controls to create all the various 7-segment digit combinations, TRK/FPA mode, light test, etc to check that the edited 7-segment font is correctly implemented.

![image](https://github.com/user-attachments/assets/1fcb5508-7ec9-414c-9e95-02aecca7229e)



## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo**, **flybywire-aircraft-a380-842 (4K)** or **flybywire-aircraft-a380-842 (8K)** download link at the bottom of the page
